room1: ['Lounge','1','1']
room2: ['Conservatory','1','2']
room3: ['BallRoom','1','3']
room4: ['DiningRoom','2','1']
room5: ['Kitchen','2','2']
room6: ['Library','2','3']
room7: ['BilliardRoom','3','1']
room8: ['Study','3','2']
room9: ['Hall','3','3']
room0: ['Corridor','0','0']
room10: ['OracleRoom','10','10']
       
   
